src/js/behaviors/picklist-transport.js

```diff
@@ -116,28 +117,34 @@ export default Behavior.extend({
   // pass 'middle' as the offsetDir to place the highlighted element in the middle
   // of the scrollable window
   /* istanbul ignore next: hard to test, but battle tested */
-  scrollTo($items, offsetDir) {
-    const $highlighted = this.getHighlighted($items);
-
-    if (!$highlighted.length || !$items.length) return;
-
-    const picklistScrollTop = this.view.$el.scrollTop();
-    const picklistHeight = this.view.$el.outerHeight();
-    const childViewHeight = $highlighted.outerHeight();
-    const childViewTop = $highlighted.position().top;
+  scrollTo($items, $scrollItem, offsetDir) {
+    if (!$items.length) return;
+    
+    if (!$scrollItem || !$scrollItem.length) {
+      $scrollItem = this.getHighlighted($items);
+    }
+ 
+    if (!$scrollItem.length) return;
+    
+    const $scrollEl = this.view.$(this.getOption('scroll'));
+
+    const picklistScrollTop = $scrollEl.scrollTop();
+    const picklistHeight = $scrollEl.outerHeight();
+    const childViewHeight = $scrollItem.outerHeight();
+    const childViewTop = $scrollItem.position().top;
     const childViewBottom = childViewTop + childViewHeight - picklistHeight;
     let offset = 0;
 
     if (offsetDir === 'middle') {
       offset = (picklistHeight / 2) - (childViewHeight / 2);
     }
-
+    
```

Comment:
extra spaces here

Reply:
my .editorconfig plugin stopped working with the my latest update :-/